masterlist.yaml

```
@@ -4310,6 +4311,18 @@ plugins:
     - <<: *patchProvided
       subs: [ 'Unofficial Fallout 3 Patch' ]
       condition: 'file("Unofficial Fallout 3 Patch.esm") and not file("EvergreenUUF3PPatch.esp")'
+    - <<: *patchProvided
+      subs: [ 'Unofficial Fallout 3 Patch' ]
+      condition: 'file("RaidersRedoneModular01.esp") and not file("RaidersRedoneEvergreenPatch.esp")'
```

Mod name is incorrect, they should placed in alphabetical order when possible, as it's easier to view for contributors.

```suggestion
    - <<: *patchProvided
      subs: [ 'Raiders Redone' ]
      condition: 'file("RaidersRedoneModular01.esp") and not file("RaidersRedoneEvergreenPatch.esp")'
    - <<: *patchProvided
      subs: [ 'Unofficial Fallout 3 Patch' ]
      condition: 'file("Unofficial Fallout 3 Patch.esm") and not file("EvergreenUUF3PPatch.esp")'
```